src/datasets/utils/resources/languages.json

```diff
@@ -861,6 +864,7 @@
     "not": "Nomatsiguenga",
     "nov": "Novial",
     "npi": "Nepali (individual language)",
+    "nqo": "N'Ko",
```

Comment:
Well done!

Reply:
Thank you for your suggestion very much.
And sorry for a lot of CI tests.